### PR TITLE
[TASK] Remove suffix from generated folder Documentation.tmpl

### PR DESCRIPTION
**Pull request by simonschaufi**
Resolves: #524

### CODE REVIEW

**Comment by simonschaufi** [comment]
@alexander-nitsche did I miss something or can you approve?

File: Documentation/Developer/Documentation.rst [hunk 1]
```diff
@@ -70,4 +68,4 @@ Publish Your Documentation
 Once you publish your extension, your documentation will automatically be rendered on docs.typo3.org.
 
 Look at the `documentation for the extension builder
-<https://docs.typo3.org/typo3cms/extensions/extension_builder/Index.html>`__ for an example.
```
Comment (alexander-nitsche) [inline Documentation/Developer/Documentation.rst:73]: What has changed in this line?

**Review by alexander-nitsche — CHANGES_REQUESTED** [review]
1) I would furthermore replace the wording "documentation template" by "documentation" in the language files.
2) Have you tested if the documentation folder really gets generated? I faced problems recently with the generation.

File: Documentation/Developer/Documentation.rst (hunk 1 above)
Comment (simonschaufi) [reply Documentation/Developer/Documentation.rst:73]: Probably new line at end of file

**Comment by simonschaufi** [comment]
There was indeed a bug in generating the docs. Fixed it with https://github.com/FriendsOfTYPO3/extension_builder/commit/1e80d62d67df4f145fb44f90090477e6e05eddc8

**Comment by alexander-nitsche** [comment]
The part
> Reverse the behavior by creating the Documentation folder by default to remove the step of renaming the folder and encourage people to write documentation. Remove the option to not create the documentation folder and instead leave it up to the developer to manually remove the folder if they do not want it.

of the issue is still missing.

**Comment by simonschaufi** [comment]
I bet that the majority of extension will never be published and are actually only for internal purposes or web agencies and they definitely don't document every single extension. I want to have a clean minimal base setup of an extension created and not a full blown extension where lots of files are created that I then need to delete again. You might have a different opinion about that but I have worked with a lot of agencies already and this is the reality.

**Comment by alexander-nitsche** [comment]
I would not provide the generated extension in the way that _is assumed_ to be the current practice of the agencies. I would provide the generated extension in the way _I would want_ agencies to develop and maintain an extension. And I would document all parts of the generated extension well in the Extension Builder documentation so that each developer can decide which parts to remove and which parts to learn - to become a better developer. This issue is about documentation, next time it will be about testing: do you know if the majority of agencies cover large parts of their private extensions' code with tests? I would not bet on it, but I would want them to cover them with tests - in the sense of a sustainable programming style.

**Comment by susannemoog** [comment]
Is there a way to make it configurable? I currently use both "Feel free to add some documentation here" (readme) and "conceived at T3Con" (test) as indicators of whether or not any 3rd-party dev used the extension builder and didn't clean up ;) Nearly every project I saw had one of the two files in their original generated form, which doesn't add value and showcases laziness... On the other hand, I understand Alexanders argument, that we would want to have the best practice case covered to show how it should be done.

**Comment by simonschaufi** [comment]
> Is there a way to make it configurable? I currently use both "Feel free to add some documentation here" (readme) and "conceived at T3Con" (test) as indicators of whether or not any 3rd-party dev used the extension builder and didn't clean up ;) Nearly every project I saw had one of the two files in their original generated form, which doesn't add value and showcases laziness... On the other hand, I understand Alexanders argument, that we would want to have the best practice case covered to show how it should be done.

Yes of course it's configurable and off by default exactly because of your reasons mentioned.

**Review by simonschaufi — COMMENTED** [review]
great

**Pull request merged**